presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMaterializedViewLogicalPlanner.java

```diff
@@ -1148,7 +1148,68 @@ public void testMaterializedViewForJoin()
             MaterializedResult viewTable = computeActual(viewQuery);
             MaterializedResult baseTable = computeActual(baseQuery);
             assertEquals(viewTable, baseTable);
-            // getExplainPlan(viewQuery, LOGICAL);
```

Comment:
I thought we had added assert plan in the previous PR?

Reply:
We did, but the diff doesn't show this here this because it's part of that PR. Unless I'm missing something?

Reply:
lets have assert plan in both places?

Reply:
I think we do, they're just in separate PRs. Once this is merged both `assertPlan`s should be in the codebase, AFAICT